internal/fs/inode/type_cache.go

```diff
@@ -15,16 +15,53 @@
 package inode
 
 import (
+	"fmt"
```

Comment:
can we move this package to internal/cache/metadata ? similar to how we did for stat cache ?